usaspending_api/api_contracts/contracts/v2/federal_accounts.md

```diff
@@ -37,6 +37,30 @@ This endpoint returns a list of federal accounts, their number, name, managing a
         + `next`: 2 (number, required, nullable)
         + `fy`: `2018` (string, required)
         + `results` (array[FederalAccountListing], fixed-type)
+ 
+# Federal Account Profile
```

Comment:
This should be in its own file.

Reply:
I was thinking that even though you pass a variable "accountNumber" on the endpoint after "federal_accounts" that it is essentially just the GET for "/api/v2/federal_accounts" that returns a specific one instead of a list. Creating a file called /v2/federal_accounts/account_number.md would lead me to believe that there is an actual route with that name and not that "account_number" is a variable.

Reply:
@tony-sappe What are your thoughts on my response to your comment?

Reply:
I don't have a good response to the URL/filename question. I suppose we need a convention to denote variables in the URL.

The obvious differentiator for me is the endpoints and responses are different which makes me believe they should be different files.